README.md

```diff
@@ -1,17 +1,15 @@
 # DoSomething.org Platform
-
-[![Production Version](http://status.dosomething.org/api/badge/us_prod.svg)](http://status.dosomething.org)
-[![Staging Version](http://status.dosomething.org/api/badge/us_staging.svg)](http://status.dosomething.org)
-[![QA Version](http://status.dosomething.org/api/badge/us_qa.svg)](http://status.dosomething.org)
-
 This is the Drupal 7 app powering [DoSomething.org](http://www.dosomething.org/) and our international affiliates.
 
+# Getting Started
+Documentation for how to get started can be found in the [wiki](https://github.com/DoSomething/dosomething/wiki).
 
-# What is DoSomething.org?
+# Frequently Asked Questions
+##### What is DoSomething.org?
 DoSomething.org makes the world suck less. The largest org for young people and social change, our 2.8 million members tackle campaigns that impact every cause, from poverty to violence to the environment to literally everything else. Any cause, anytime, anywhere. *mic drop*
```

Comment:
While we're in here, we should kill this number. It will always be out of date. I'd recommend we just kill this whole section since it can be found on the website...